cmd/agentctl/cli/flags.go

```diff
@@ -43,11 +42,9 @@ type ClientOptions struct {
 	PortHTTP     int
 	ServiceLabel string
 	Endpoints    []string
+	TLS          bool
 
-	// TODO: support TLS
-	TLS        bool
-	TLSVerify  bool
-	TLSOptions *tlsconfig.Options
```

Comment:
These options will be needed for providing security related options for connection (gRPC) to the agent and should not be used for the KVDB connection.

Reply:
So grpc is the only thing which can be configured via flags? Right now I'm using `--tls` flag to tell whether or not use TLS for different connections, for example if all files (cert, key and ca) are defined for kvdb in config file, but I want to connect to local etcd, so I don't need any TLS for now, but later I want to connect to remote etcd with auth enabled so I going to need those files. 
I'll rethink this with clear head tomorrow. Now I'm only about to push today's work and go home :relieved:

Reply:
Well, I guess we can keep this shared for now since the agentctl uses one connection per single run. However later when we start using viper any config option defined for configuration can be overriden automatically from [env vars](https://github.com/spf13/viper#working-with-environment-variables) and [flags](https://github.com/spf13/viper#working-with-flags) by using `BindEnv` and `BindFlags`.